checkout/other-features.md

```diff
@@ -1353,6 +1166,8 @@ The structure of a problem message will look like this:
 
 {% include settlement-reconciliation.md api_resource="paymentorders" %}
 
+{% include payment-menu-events.md api_resource="paymentorders" %}
```

Comment:
Considering this include is used in both payment instruments and checkout, I think we should go for a more generic name. I think `seamless-view-events` sound more generic than `payment-menu-events`. Can you rename it?

```suggestion
{% include seamless-view-events.md api_resource="paymentorders" %}
```